arkouda/alignment.py

```diff
@@ -278,20 +278,23 @@ def in1d_intervals(vals, intervals, symmetric=False):
         return found
 
 
-def search_intervals(vals, intervals):
+def search_intervals(vals, intervals, tiebreak=None):
     """
-    Given an array of query vals and non-overlapping, half-open (pythonic)
-    intervals, return the index of the interval containing each query value,
-    or -1 if not present in any interval.
+    Given an array of query vals and half-open (pythonic) intervals, return the index of the
+    best (see tiebreak) interval containing each query value, or -1 if not present in any
+    interval.
 
     Parameters
     ----------
     vals : (sequence of) pdarray(int, uint, float)
         Values to search for in intervals. If multiple arrays, each "row" is an item.
     intervals : 2-tuple of (sequences of) pdarrays
-        Non-overlapping, half-open intervals, as a tuple of
-        (lower_bounds_inclusive, upper_bounds_exclusive)
-        Must have same shape/dtype as vals.
+        Half-open intervals, as a tuple of (lower_bounds_inclusive, upper_bounds_exclusive).
+        Must have same dtype(s) as vals.
+    tiebreak : (optional) pdarray, numeric
+        When a value is present in more than one interval, the interval with the
+        lowest tiebreak value will be chosen. If no tiebreak is given, the
+        first containing interval will be chosen.
 
     Returns
     -------
```

Comment:
would it be worthwhile to add an example demonstrating how this works with and without `tiebreak`? I think a slimmed down version of the test would be great